We need to disable this, or else memmove will not work properly in Hurd's glibc.

From 69f807d6a4afa43e7a596e04f22923567b235ce9 Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Wed, 29 Jul 2015 10:34:01 +0200
Subject: [PATCH] Disable the fix for now, the resulting code seems bogus

---
 .topmsg                 | 11 +++++++++++
 sysdeps/mach/pagecopy.h |  3 +++
 2 files changed, 14 insertions(+)

diff --git a/sysdeps/mach/pagecopy.h b/sysdeps/mach/pagecopy.h
index 29b73ce..9d1f4de 100644
--- a/sysdeps/mach/pagecopy.h
+++ b/sysdeps/mach/pagecopy.h
@@ -20,7 +20,10 @@
 
 /* Threshold at which vm_copy is more efficient than well-optimized copying
    by words.  This parameter should be tuned as necessary.  */
+#define PAGE_THRESHOLD		(16384)	/* XXX Tune this.  */
+#if 0
 #define PAGE_COPY_THRESHOLD		(16384)	/* XXX Tune this.  */
+#endif
 
 #define PAGE_SIZE		__vm_page_size
 #define PAGE_COPY_FWD(dstp, srcp, nbytes_left, nbytes)			      \
-- 
2.4.6